### DISCUSSION TITLE
Failed to download resource "libusb" for Apple Silicon

### DISCUSSION THREAD

**Discussion by Leon2xiaowu** [Homebrew/discussions#2086] [Getting started] [answered]
when i install `libusb`,  the resource while be 403. 

```
~  brew install libusb
==> Downloading https://homebrew.bintray.com/bottles/libusb-1.0.24.arm64_big_sur.bottle.ta
#=#=-#  #                                                                     
curl: (22) The requested URL returned error: 403 Forbidden
Error: Failed to download resource "libusb"
Download failed: https://homebrew.bintray.com/bottles/libusb-1.0.24.arm64_big_sur.bottle.tar.gz
```

**Comment by nandahkrishna** [comment, marked as answer]
Bintray has shut down. Your Homebrew installation probably hasn't been updated in a long time. You will have to `brew update` to be able to download from GitHub Packages, which is where bottles are now hosted. (Your `brew config` and `brew doctor` information will help confirm this.)

**Reply by Leon2xiaowu** [reply]
Thanks, the problem is solved after the upgrade.
